conf/app.routes

```diff
@@ -2,6 +2,7 @@
 POST       /tax-check                           uk.gov.hmrc.hec.controllers.TaxCheckController.saveTaxCheck
 
 POST       /match-tax-check                     uk.gov.hmrc.hec.controllers.TaxCheckController.matchTaxCheck
+GET        /tax-check-codes                     uk.gov.hmrc.hec.controllers.TaxCheckController.getUnexpiredTaxCheckCodes
```

Comment:
might change this to `/unexpired-tax-check-codes`

Reply:
maybe `unexpired-tax-checks` to make it more inclusive of the other bits of info it's returning?

Reply:
updated - https://github.com/hmrc/hec/pull/18/commits/cdc197f9152bf39122ff6dbc8425330fb6537cb6